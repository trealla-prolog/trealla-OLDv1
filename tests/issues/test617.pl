:- op(300,xfx,\\).

main :-
	read_from_chars("arg(1,(\\) \\\\ '', Y).", X),
	read_from_chars("arg(1,\\ \\\\ '', Y).", X).

:- initialization(main).
