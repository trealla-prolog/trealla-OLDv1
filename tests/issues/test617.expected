   error(syntax_error(operator_clash),read_term_from_chars/3).
